Applications/Spire/Include/Spire/Ui/SideInputWidget.hpp

```diff
@@ -0,0 +1,35 @@
+#ifndef SPIRE_SIDE_INPUT_WIDGET_HPP
+#define SPIRE_SIDE_INPUT_WIDGET_HPP
+#include <QWidget>
+#include "Nexus/Definitions/Side.hpp"
+#include "Spire/Spire/Spire.hpp"
+
+namespace Spire {
+
+  //! Represents a widget for selecting Side values.
+  class SideInputWidget : public QWidget {
+    public:
+
+      //! Signals that a Side was selected.
+      /*!
+        \param side The selected Side.
+      */
+      using SideSignal = Signal<void (const Nexus::Side& side)>;
```

Comment:
A signal indicates an action was performed and hence should be named after a verb. ```UpdateSignal``` is more descriptive than ```SideSignal```.

Reply:
Pass ```Nexus::Side``` by value, refer to:

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-conventional